### ISSUE TITLE
Exported file fails to import (various wallets)

### ISSUE THREAD

**Issue by centrd** [aaroncarpenter/fork-board-wallet-tool#2]
- fork-board-tool release 0.1.0
- ForkBoard installed from ForkBoard-1.0.1-setup
- Test machine running Windows 10 Pro
- Installed full nodes: Chia, Flax, SIT, HDDCoin, Skynet.
- This is a non-virtualised setup.

Exported 1) All wallets; 2) each fork one by one.

On attempting to import each one of those files, this message briefly shows up, and import appears to fail:

> Invalid file format detected attempting to import the wallet tool file from [....]

Exported json files look like this:
```json
{
         "name": "ForkBoard Wallet Tool Export File",
         "date": "12/24/2021, 2:15:09 AM",
         "forks": "HDDcoin",
         "walletConfiguration": [
	{
		"wallet": "hdd1...."
	}
]
      }
```

**Comment by aaroncarpenter** [comment]
Well, so since you got this far, it means that the tool worked. So, it would seem the white screen issue to be isolated to virtualized windows environments. ForkBoard validates the input file and I tweaked the schema that the wallet tool exports. Thought I made the corresponding change in ForkBoard but perhaps it didn't make it into the 1.0.1 prerelease build. I'll be back from vacation on Wednesday and will republish the build.
